server/pulp/server/async/tasks.py

```diff
@@ -25,7 +25,10 @@
 from pulp.server.managers.repo import _common as common_utils
 from pulp.server.managers import factory as managers
 from pulp.server.managers.schedule import utils
+from pulp.server import config
 
+if config.get('profiling', 'enabled') is True:
```

Comment:
Can we unguard this import statement and move it up with the other standard imports? We can import it 100 percent of the time I think.